---
title: "Published Work"
permalink: /publishedwork/
author_profile: true
redirect_from:
  - /publications
---

{% include base_path %}
[**Deep neural networks with knockoff features identify nonlinear causal relations and estimate effect sizes in complex biological systems**](https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giad044/7217083)  
**Citation:** Fan, Z., Kernan, K. F., **Sriram, A.**, Benos, P. V., Canna, S. W., Carcillo, J. A., Kim, S., & Park, H. J. (2023). Deep neural networks with knockoff features identify nonlinear causal relations and estimate effect sizes in complex biological systems. GigaScience, 12. https://doi.org/10.1093/gigascience/giad044 

<img src="https://github.com/ads303/ads303.github.io/assets/108133717/7d897a78-2ed2-461d-b818-0230e96d50c6.png" width="280" height="280">

**Abstract:** Learning the causal structure helps identify risk factors, disease mechanisms, and candidate therapeutics for complex diseases. However, although complex biological systems are characterized by nonlinear associations, existing bioinformatic methods of causal inference cannot identify the nonlinear relationships and estimate their effect size. To overcome these limitations, we developed the first computational method that explicitly learns nonlinear causal relations and estimates the effect size using a deep neural network approach coupled with the knockoff framework, named causal directed acyclic graphs using deep learning variable selection (DAG-deepVASE). Using simulation data of diverse scenarios and identifying known and novel causal relations in molecular and clinical data of various diseases, we demonstrated that DAG-deepVASE consistently outperforms existing methods in identifying true and known causal relations. In the analyses, we also illustrate how identifying nonlinear causal relations and estimating their effect size help understand the complex disease pathobiology, which is not possible using other methods. With these advantages, the application of DAG-deepVASE can help identify driver genes and therapeutic agents in biomedical studies and clinical trials.

[**Refining empiric subgroups of pediatric sepsis using machine-learning techniques on observational data**](https://www.frontiersin.org/articles/10.3389/fped.2023.1035576/full)  
**Citation:** Qin, Y., Caldino Bohn, RI., **Sriram, A.**, Kernan K., Carcillo, JA., Kim, S., Park, HJ. Refining empiric subgroups of pediatric sepsis using machine-learning techniques on observational data. Frontiers in Pediatrics 11 (2023). https://doi.org:10.3389/fped.2023.1035576

<img src="https://user-images.githubusercontent.com/108133717/233152690-831aa9b2-1703-4f89-9a98-cb7e125e7742.png" width="280" height="280">

**Abstract:** Sepsis contributes to 1 of every 5 deaths globally with 3 million per year occurring in children. To improve clinical outcomes in pediatric sepsis, it is critical to avoid “one-size-fits-all” approaches and to employ a precision medicine approach. To advance a precision medicine approach to pediatric sepsis treatments, this review provides a summary of two phenotyping strategies, empiric and machine-learning-based phenotyping based on multifaceted data underlying the complex pediatric sepsis pathobiology. Although empiric and machine-learning-based phenotypes help clinicians accelerate the diagnosis and treatments, neither empiric nor machine-learning-based phenotypes fully encapsulate all aspects of pediatric sepsis heterogeneity. To facilitate accurate delineations of pediatric sepsis phenotypes for precision medicine approach, methodological steps and challenges are further highlighted.

[**Genetic Risk Factors Associated with SARS-CoV-2 Susceptibility in Multiethnic Populations**](https://www.iomcworld.org/articles/genetic-risk-factors-associated-with-sarscov2-susceptibility-in-multiethnic-populations-93385.html)  
**Citation:** **Sriram AD**, Kerr K. Genetic Risk Factors Associated with SARS-CoV-2 Susceptibility in Multiethnic Populations. J Biol Today's World, 2022, 11(4), 001-004.  
<img src="https://user-images.githubusercontent.com/108133717/231258716-0dce9f4c-320b-4f5a-8ce8-45d27faa6bc4.png" width="280" height="280">

**Abstract:** Susceptibility to infection from severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes the disease COVID-19, may be understood more clearly by looking at genetic variants and their associations to susceptibility phenotype. I conducted a genome-wide association study of SARS-CoV-2 susceptibility in a multiethnic set of three populations (European, African, and South Asian) from a UK BioBank clinical and genomic dataset. I estimated associations between susceptibility phenotype and genotyped or imputed SNPs, adjusting for age at enrollment, sex, and the ten top principal components of ancestry. Three genome-wide significant loci and their top associated SNPs were discovered in the European ancestry population: SLC6A20 in the chr3p21.31 locus (rs73062389-A; P=2.315 × 10-12), ABO on chromosome 9 (rs9411378-A; P=2.436 × 10-11) and LZTFL1 on chromosome 3 (rs73062394; P=4.4 × 10-11); these SNPs were not found to be significant in the African and South Asian populations. A multiethnic GWAS may help elucidate further insights into SARS-CoV-2 susceptibility.

[**Assessment of Autism Zebrafish Mutant Models Using a High-Throughput Larval Phenotyping Platform**](https://www.frontiersin.org/articles/10.3389/fcell.2020.586296/full)  
**Citation:** Colón-Rodríguez, A., Uribe-Salazar, J. M., Weyenberg, K. B., **Sriram, A.**, Quezada, A., Kaya, G., Jao, E., Radke, B., Lein, P. J., Dennis, M. Y., (2020). Assessment of Autism Zebrafish Mutant Models Using a High-Throughput Larval Phenotyping Platform. Frontiers in Cell and Developmental Biology, 8.

<img src="https://user-images.githubusercontent.com/108133717/195214144-cc665650-fcc0-4617-9222-91a27a422078.png" width="300" height="300">   

**Abstract:** In recent years, zebrafish have become commonly used as a model for studying human traits and disorders. Their small size, high fecundity, and rapid development allow for more high-throughput experiments compared to other vertebrate models. Given that zebrafish share >70% gene homologs with humans and their genomes can be readily edited using highly efficient CRISPR methods, we are now able to rapidly generate mutations impacting practically any gene of interest. Unfortunately, our ability to phenotype mutant larvae has not kept pace. To address this challenge, we have developed a protocol that obtains multiple phenotypic measurements from individual zebrafish larvae in an automated and parallel fashion, including morphological features (i.e., body length, eye area, and head size) and movement/behavior. By assaying wild-type zebrafish in a variety of conditions, we determined optimal parameters that avoid significant developmental defects or physical damage; these include morphological imaging of larvae at two time points [3 days post fertilization (dpf) and 5 dpf] coupled with motion tracking of behavior at 5 dpf. As a proof-of-principle, we tested our approach on two novel CRISPR-generated mutant zebrafish lines carrying predicted null-alleles of syngap1b and slc7a5, orthologs to two human genes implicated in autism-spectrum disorder, intellectual disability, and epilepsy. Using our optimized high-throughput phenotyping protocol, we recapitulated previously published results from mouse and zebrafish models of these candidate genes. In summary, we describe a rapid parallel pipeline to characterize morphological and behavioral features of individual larvae in a robust and consistent fashion, thereby improving our ability to better identify genes important in human traits and disorders.

[**Abstract 109: Biological Pathways Enriched For Genes Associated With Lesion Burden In Cerebral Cavernous Malformation**](https://www.ahajournals.org/doi/10.1161/str.53.suppl_1.109)  
**Citation:** **Sriram, A.**, Weinsheimer, S., Nelson, J., Mabray, M., Zabramski, J. M., Akers, A., Hart, B., Morrison, L., McCulloch, C. E., Kim, H., (2022). Abstract 109: Biological Pathways Enriched For Genes Associated With Lesion Burden In Cerebral Cavernous Malformation. Stroke, (Suppl_1), A109–A109.

<img src="https://user-images.githubusercontent.com/108133717/231259251-57e64768-ff86-4056-a573-834ee726df2c.png" width="300" height="400"> 

**Abstract:** Familial Cerebral Cavernous Malformation (CCM) is an autosomal dominant disorder caused by mutations in *KRIT1*, *CCM2* and *PDCD10*. CCMs can result in seizures, neurological deficits, and intracranial hemorrhage. Our prior research identified genetic modifiers that contribute to CCM progression. We hypothesized that genes associated with CCM progression (lesion burden, large lesion size, and hemorrhage) are enriched in biological pathways. We performed a functional enrichment analysis of top associated genes from a genome-wide panel to identify pathways that, when dysregulated, may contribute to CCM progression.


